src/components/radio-button/radio-button.e2e.ts

```diff
@@ -272,6 +285,17 @@ describe("calcite-radio-button", () => {
 
     expect(focusEvent).toHaveReceivedEventTimes(1);
     expect(blurEvent).toHaveReceivedEventTimes(1);
+
+    //in addition to .click() check with .callMethod("click") as well
```

Comment:
I don't think we need to assert on focus/blur events. The click toggling `checked` should suffice.